.github/workflows/ci.yml

```diff
@@ -23,13 +23,20 @@ jobs:
         run: yarn install --frozen-lockfile
       - name: Lint
         run: yarn lint
-      - name: Notify deploy failed
-        if: ${{ failure() && github.ref == 'refs/heads/master' }}
```

Comment:
https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#about-contexts-and-expressions